src/Microsoft.VisualStudio.Editor.Razor/BackgroundParser.cs

```diff
@@ -343,12 +347,24 @@ private void WorkerLoop()
                 }
             }
 
-            private RazorCodeDocument ParseChange(ITextSnapshot snapshot, CancellationToken token)
+            private RazorCodeDocument ParseChange(ITextSnapshot snapshot, RazorCodeDocument previousResult, CancellationToken token)
             {
                 EnsureOnThread();
 
                 var sourceDocument = new TextSnapshotSourceDocument(snapshot, _filePath);
-                var imports = _templateEngine.GetImports(_filePath);
+
+                IEnumerable<RazorSourceDocument> imports = null;
+
+                try
+                {
+                    imports = _templateEngine.GetImports(_filePath);
+                }
+                catch (IOException)
```

Comment:
What if instead of this silliness we used different options when creating the stream?

http://index/?leftProject=Microsoft.CodeAnalysis.Workspaces&leftSymbol=bjlw2xzo5g0k&rightProject=Microsoft.CodeAnalysis.Workspaces.Desktop&file=Workspace%5CFileTextLoader.cs&line=171

VS

https://msdn.microsoft.com/en-us/library/system.io.file.openread(v=vs.110).aspx#Anchor_2